cabal-docspec/MANUAL.md

```diff
@@ -83,11 +83,32 @@ However, in this list we mostly only list and show the --option version of them.
 :   Message to return when the evaluation is timed out.
     Default is **\* Hangs forever \***.
 
+**\--ghci-rtsopts** *options*
+
+:   RTS options for GHCi process
+
+**\--skip-properties**
```

Comment:
Is `--skip-properties` effectively the same thing as `--no-simple-properties`? Perhaps it would be cleaner to just have one flag rather than two. (I'm not sure what happens if you pass both `--skip-properties` and `--simple-properties`.)

Reply:
Fair. In theory there could be third way to handle properties (more faithful to `doctest` e.g.) but I don't know.

Reply:
I'll change the flags to `--check-properties` and `--no-check-properties` with latter being the default.